### PR TITLE
BTAT-5870: Updated VAT Return page to show outstanding amount

### PR DESCRIPTION
**Pull request by davidmthornton**

### CODE REVIEW

File: test/views/templates/returns/PaymentStatusTemplateSpec.scala [hunk 1]
```diff
@@ -24,21 +24,27 @@ import views.templates.TemplateBaseSpec
 
 class PaymentStatusTemplateSpec extends TemplateBaseSpec {
 
+  val whatYouOweCalc = "We have worked out what you owe based on any payments you might have made on your account. "
```
Comment (lewishay) [inline test/views/templates/returns/PaymentStatusTemplateSpec.scala]: Could you one new test in this spec, for when an outstanding amount is something different than return total amount (i.e. a partial payment has been made)? Just to prove the content is rendering correctly.

File: app/services/ReturnsService.scala [hunk 2]
```diff
@@ -33,81 +34,79 @@ class ReturnsService @Inject()(vatObligationsConnector: VatObligationsConnector,
...
-}
-
-  def getDirectDebitStatus(vrn: String)(implicit hc: HeaderCarrier, ec: ExecutionContext): Future[ServiceResponse[Boolean]] = {
-  financialDataConnector.getDirectDebitStatus(vrn) map {
-  case Right(directDebitStatus) => Right(directDebitStatus)
-  case Left(_) => Left(DirectDebitStatusError)
-}
-}
+    val moneyOwed = payment.fold(true)(_.outstandingAmount != 0)
+    val oweHmrc: Option[Boolean] = payment map {
```
Comment (lewishay) [inline app/services/ReturnsService.scala]: Do we need this sort of logic around checking charge types? Now that we know the outstanding amount will only ever be negative when it's a repayment, we can use that figure to work this out and not have to care about charge type names.

File: test/views/templates/returns/PaymentStatusTemplateSpec.scala (hunk 1 above)
Comment (davidmthornton) [reply test/views/templates/returns/PaymentStatusTemplateSpec.scala]: Yes thanks for pointing that out.

File: app/services/ReturnsService.scala (hunk 2 above)
Comment (davidmthornton) [reply app/services/ReturnsService.scala]: Thanks that's true.

**Pull request merged**